executor/logger/worker.go

```diff
@@ -109,12 +122,15 @@ func (w *Worker) sendKafkaEvent(logReq LogRequest) error {
 		{Key: NamespaceAttr, Value: []byte(w.Namespace)},
 		{Key: EndpointAttr, Value: []byte(w.PredictorName)},
 	}
-
+	w.Log.Info("kafkaHeaders is", "kafkaHeaders", kafkaHeaders)
 	err = w.Producer.Produce(&kafka.Message{
 		TopicPartition: kafka.TopicPartition{Topic: &w.KafkaTopic, Partition: kafka.PartitionAny},
 		Value:          *logReq.Bytes,
 		Headers:        kafkaHeaders,
 	}, nil)
+	w.Log.Info("Topic Partition:", "topic", kafka.TopicPartition{Topic: &w.KafkaTopic, Partition: kafka.PartitionAny})
+	w.Log.Info("Value:", "logReqBytes", *logReq.Bytes)
+	w.Log.Info("Kafka Headers:", "headers", kafkaHeaders)
```

Comment:
:upside_down_face: This duplicates the headers-logging on line 125 above - no need for both :)